common/jvm/src/test/scala/com/redi/scalademo/business/NumericStringValidatorTest.scala

```diff
@@ -0,0 +1,38 @@
+package com.redi.scalademo.business
+
+import org.scalatest.{Matchers, path}
+
+class NumericStringValidatorTest extends path.FunSpec with Matchers {
```

Comment:
> Business logic shouldn't fail based on your compilation platform.

I think whether you can think like that or not depends on how much you trust on compilation for each platform. For this project, I think we should run them on both to give the audience additional confidence.

Putting the test on shared project also communicates to other developers that the test can and should be run on both JS and JVM. If you see tests under either JS or JVM project, it is usually because there is some JS or JVM specific thing you want to test, but not because developer thought the code *should* behave the same in either platform.